app/code/core/Mage/Cron/Model/Schedule.php

```diff
@@ -107,8 +107,8 @@ public function trySchedule($time)
             && $this->matchCronExpression($e[4], $d['wday']);
 
         if ($match) {
-            $this->setCreatedAt(strftime('%Y-%m-%d %H:%M:%S', time()));
-            $this->setScheduledAt(strftime('%Y-%m-%d %H:%M', (int)$time));
+            $this->setCreatedAt(date(Varien_Db_Adapter_Pdo_Mysql::TIMESTAMP_FORMAT));
+            $this->setScheduledAt(date('Y-m-d H:i:00', (int)$time));
```

Comment:
```suggestion
            $this->setScheduledAt(date('Y-m-d H:i', (int)$time));
```

Reply:
This is strange to put a value without seconds for me, even if this will probably works.
```
php > echo date('Y-m-d H:i');
2023-01-14 20:49
php > echo date('Y-m-d H:i:00');
2023-01-14 20:49:00
```

Reply:
I just compared before/after and this is the only change i found in output. (y, its strange ;) )

Reply:
Do you agree to keep current behavior?

Reply:
Either way both will save the value with 00 seconds

Reply:
I don't have an opinion on this so as you all prefer :-)